pydropsonde/circles.py

```diff
@@ -15,6 +23,172 @@ class Circle:
     platform_id: str
     segment_id: str
 
-    def dummy_circle_function(self):
-        print(self.flight_id, self.segment_id)
+    def get_xy_coords_for_circles(self):
```

Comment:
the changes that are already part of another pull request should not be part of this one as well